visualization/app/codeCharta/util/scenarioHelper.ts

```diff
@@ -49,18 +49,22 @@ export class ScenarioHelper {
 		return scenarioItems
 	}
 
-	private static isScenarioAppliable(scenario: RecursivePartial<Scenario>, metricData: NodeMetricData[]) {
-		if (scenario.area && !metricData.find(x => x.name === scenario.area.areaMetric)) {
-			return false
-		}
-		if (scenario.color && !metricData.find(x => x.name === scenario.color.colorMetric)) {
-			return false
-		}
-		if (scenario.height && !metricData.find(x => x.name === scenario.height.heightMetric)) {
-			return false
+	private static isScenarioAppliable(scenario: RecursivePartial<Scenario>, metricData: MetricData) {
```

Comment:
```suggestion
	private static isScenarioApplicable(scenario: RecursivePartial<Scenario>, metricData: MetricData) {
```

Seems like this needs some other updates as well :)